src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,41 @@
+package duke.task;
+
+/**
+ * Task to be done and inserted into the Duke program.
+ * CS2103T iP
+ * AY22/23 Semester 1
+ *
+ * @author Perry Wong
+ */
+public class Task {
+
+    protected String description;
+
+    protected boolean isDone;
```

Comment:
I like the way you name this boolean attribute!

Reply:
Thanks!